### ISSUE TITLE
Cannot handle uint64

### ISSUE THREAD

**Issue by ivanjaros** [goccy/go-json#419]
Just like any other json un/marshaler, when u/int64 is presetned in string form, it will fail to unmarshal it onto uint64 number.
```
package main

import (
	"fmt"
	"github.com/goccy/go-json"
)

type Message struct {
	Size uint64 `json:"size"`
}

func main() {
	var msg Message 
	fmt.Println(json.Unmarshal([]byte(`{"size": "25"}`), &msg))
	println(msg.Size)
}

json: cannot unmarshal number " into Go value of type uint64
0
```

**Comment by ivanjaros** [comment]
OMG, by accident I found the solution in another easyjson issue:
```
type Message struct {
	Size uint64 `json:"size,string"` <---- ,string appendix
}
``

and **it works**! This should be stated in readme with big bold letters!!

**Comment by ivanjaros** [comment]
OK, so i found that the issue is javascript. As long as a number is present in json unquoted, it will be un/marshalled as such. This library therefore works properly because go does not have uint64 limitation that JS does, so unless the number in js does not go above the limit, which can be mitigated with BigInt() it can be handled as number and not as string.

tl;dr send number, parse number or send string and parse string, not both.
